sagemaker-autopilot/housing_prices/autopilot_california_housing.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>On second thoughts, please uncomment these. I think it's safer to delete resources once done as a default, so that even if users don't notice it while running the notebook, they accidentally don't incur additional charges. They can comment it if need be later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>